client/src/plus/offer-overview/offer-overview-subscribe/index.tsx

```diff
@@ -252,6 +252,13 @@ function OfferOverviewSubscribe() {
           <OfferDetails offerDetails={PLUS_10} period={period}></OfferDetails>
         </div>
       </section>
+      <p className="plus-for-companies">
+        * Do you need MDN Plus for your company?{" "}
```

Comment:
Just checking: This is the only asterisk on the page, right, or was there supposed to be a corresponding asterisk somewhere above?

Reply:
Yup